src/ir/proto/sql/sql_ir.proto

```diff
@@ -49,16 +49,16 @@ message MergeOperation {
 
 // The kind of a tag.
 enum TagKind {
-  UNKNOWN = 0;
-  INTEGRITY = 1;
-  CONFIDENTIALITY = 2;
+  TK_UNKNOWN = 0;
```

Comment:
Use full name `TAG_KIND_UNKNOWN`, ...

Reply:
Sounds good, will do for this and the other.

Reply:
Done.